docs/_articles/en/release-information.md

```diff
@@ -3,6 +3,38 @@ title: Release Information
 lang: en
 ---
 
+## [v2.4.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.4.0) (11-04-2020)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.3.0...v2.4.0)
+
+### Release Summary
+
+* RetireJS - Supports a new engine to detect vulnerable Javascript Library in the project
+* Underlying PMD Engine is upgraded to 6.29.0 (from 6.28.0)
+* Bug fixes 
+
+**Closed issues:**
+
+- Missing meta mime types detections [\#270](https://github.com/forcedotcom/sfdx-scanner/issues/270)
+- @W-8118474@ specifying --category flag in scanner:run command has no effect for JavaScript and TypeScript files [\#224](https://github.com/forcedotcom/sfdx-scanner/issues/224)
```

Comment:
I think we need a process change. I closed this issue after 2.3.0 was released, but it was fixed in 2.3.0. Not sure if it's best to leave this in, or take it out.

Reply:
Hmm, I see your point. The closed date and when tags are created. So, looks like we need to close the issues before we generate release notes and before release tags are created. May be next time?